### PR TITLE
Add some hunter equipment

### PR DESCRIPTION
**Pull request by podnov**
Add Trueaim Gauntlets

Add Bone Slicing Hatchet

### CODE REVIEW

**Comment by podnov** [comment]
> The art for these will need added as well

Do you mean that I'm missing something? If so, I'm not certain I understand what I'm missing.

**Comment by timhul** [comment]
Hi, what IsaacSmith13 is referring to is that while the art for the gauntlets exist the icon does not. QML needs to compile the .png files such that it has art to display for each icon.

You do this by:

1. Finding the appropriate icon on [wikis like this one](https://wowwiki.fandom.com/wiki/Category:WoW_Icons:_Inventory) and placing it in the appropriate QML/Assets directory in the repository (in this case under QML/Assets/items/).
2. Add the file to git, which is set up to handle images using Git LFS.
3. Add it to qml.qrc either manually editing or via Qt Creator (I prefer Qt Creator). If doing it via Qt Creator you expand Resources in the project, right-click qml.qrc -> Add Existing Directory -> point towards QML/Assets/items. It will warn you it failed to add many files as they already exist: ignore this warning. If you re-compile you'll see it will recompile qml.qrc, and git status will show that qml.qrc was modified.

File: Equipment/EquipmentDb/Weapons/1hAxes.xml [hunk 1]
```diff
@@ -1,5 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no" ?>
 <weapons>
+	<melee_weapon id="18737" phase="1">
+		<info name="Bone Slicing Hatchet" type="AXE" slot="1H" unique="no" req_lvl="57" item_lvl="62" quality="RARE" boe="no" icon="Inv_axe_11.png"/>
+		<dmg_range min="48" max="90" speed="1.7"/>
+		<stats>
+			<stat type="AGILITY" value="15"/>
```
Comment (timhul) [inline Equipment/EquipmentDb/Weapons/1hAxes.xml]: It should have 13 Agility.

Comment (podnov) [reply Equipment/EquipmentDb/Weapons/1hAxes.xml]: Good catch on the type-o, this should be fixed now.

Comment (podnov) [reply Equipment/EquipmentDb/Weapons/1hAxes.xml]: This gtg now?

Comment (timhul) [reply Equipment/EquipmentDb/Weapons/1hAxes.xml]: Yes. I apologize for the delay, I've been away for Christmas vacation.

**Comment by podnov** [comment]
No, thank you!

**Pull request merged**